tools/schemacode/bidsschematools/tests/test_expressions.py

```diff
@@ -77,3 +86,67 @@ def test_checks(schema_obj):
 def test_expected_failures(expr):
     with pytest.raises(ParseException):
         expression.parse_string(expr)
+
+
+def test_valid_sidecar_field(schema_obj):
```

Comment:
I sort of totally improvised testing this, so there may be a simpler way to do this.